src/Akka.Persistence.Sql.Linq2Db.Tests/SqlServerJournalPerfSpec.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using Akka.Configuration;
+using Akka.Persistence.TestKit.Performance;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.Persistence.Sql.Linq2Db.Tests
+{
+    [Collection("SqlServerSpec")]
+    public class SqlServerJournalPerfSpec : JournalPerfSpec
+    {
+        private static Config InitConfig(SqlServerFixture fixture)
+        {
+            // need to make sure db is created before the tests start
+            DbUtils.Initialize(fixture.ConnectionString);
+            
+            var specString = $@"
+                akka.persistence {{
+                    publish-plugin-commands = on
+                    journal {{
+                        plugin = ""akka.persistence.journal.linq2db""
+                        linq2db {{
+                            class = ""{typeof(Linq2DbWriteJournal).AssemblyQualifiedName}""
+                            plugin-dispatcher = ""akka.actor.default-dispatcher""
+                            provider-name = SqlServer
```

Comment:
SQL Server Version should be As specific as possible based on one of the values in `Linq2Db.ProviderName`. Otherwise Provider detection has to run every time connection is open (and has an impact on performance.)

For Docker tests, "SqlServer2017" can be used.